### ISSUE TITLE
Hyper-Paramter Tuning

### ISSUE THREAD

**Issue by rematchka** [RanaMostafaAbdElMohsen/Traffic_Sign_Recognition#4]
- Batch Size

-  Learning Rate

-  Use Decay Learning rate

- Use Ada-delta as optimizers

- Initialization of the network

- Callbacks: Early stopping
